src/lib/platform/OSXScreen.mm

```diff
@@ -811,6 +811,8 @@
 			IORegistryEntrySetCFProperty(entry, CFSTR("IORequestIdle"), kCFBooleanFalse);
 			IOObjectRelease(entry);
 		}
+		IOPMAssertionID assertionID; 
```

Comment:
nitpick, extra whitespace at end of line

Reply:
Resolved in latest force-push and rebase.